src/classes/BDI_DataImportService.cls

```diff
@@ -728,6 +751,11 @@ global with sharing class BDI_DataImportService {
         DescribeFieldResult dfr = UTIL_Describe.getFieldDescribe(dataImportObjectName, sourceFieldName);
         String targetField = dfr.inlineHelpText != null ? dfr.inlineHelpText.split('\\.')[1] : null;
 
+        // Donation Campaign is a special case because its mapping is hard-coded, not in help text
```

Comment:
How is this handled for Contact1Imported, Contact2Imported, etc? Those are all hard-coded somewhere, right?

Reply:
Great question. I'm not sure there's a central spot for all of the lookups because of how they're used historically as outputs, not inputs. I'll look around a bit. Because we're making Account1Imported and Contact1Imported always available, we're handling them elsewhere for now, but this is certainly something to look at at *least* when/if we get to offering more of those lookups as inputs.

Reply:
Removed, since we are treating Donation Campaign as a normal mapped field now with help text mapping.